src/ray/gcs/accessor.h

```diff
@@ -560,6 +560,20 @@ class NodeInfoAccessor {
   /// \return Status
   virtual Status AsyncReSubscribe() = 0;
 
+  /// Set the Intrenal Config string from the first node.
+  ///
+  /// \param config Map of config options
+  /// \return Status
+  virtual Status SetInternalConfig(
+      std::unordered_map<std::string, std::string> config) = 0;
```

Comment:
```suggestion
      std::unordered_map<std::string, std::string> &config) = 0;
```

Probably don't want to copy this unnecessarily